universal-login-ops/src/dev/startDevelopment.ts

```diff
@@ -3,15 +3,15 @@ import {getWallets} from 'ethereum-waffle';
 import {providers, Wallet} from 'ethers';
 import {ContractWhiteList, getContractHash, SupportedToken, ContractJSON, ETHER_NATIVE_TOKEN, UNIVERSAL_LOGIN_LOGO_URL} from '@universal-login/commons';
 import {RelayerClass, Config} from '@universal-login/relayer';
-import {beta2} from '@universal-login/contracts';
+import {gnosisSafe} from '@universal-login/contracts';
 import {mockContracts} from '@universal-login/contracts/testutils';
 import {ensureDatabaseExist} from '../common/ensureDatabaseExist';
 import {startDevelopmentRelayer} from './startRelayer';
 import {startGanache} from './startGanache';
 import {deployENS} from './deployEns';
-import deployWalletContractOnDev from './deployWalletContractOnDev';
+import {deployGnosisSafe} from './deployWalletContractOnDev';
 import deployToken from './deployToken';
-import deployFactory from '../ops/deployFactory';
+import {deployFactoryGnosis} from '../ops/deployFactory';
```

Comment:
Should this method be included in {gnosisSafe} from '@universal-login/contracts' package as deployFactory?

Reply:
It might be... But deployFactoryGnosis is ops function and like the other ops deployment functions, has some comments informing about progress and stuff typical for that kind of functions.